manifests/o/OpenRCT2/OpenRCT2/0.4.0/OpenRCT2.OpenRCT2.installer.yaml

```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: OpenRCT2.OpenRCT2
+PackageVersion: 0.4.0
```

Comment:
```suggestion
PackageVersion: 0.4.0master-c6302a1
```

Reply:
Makes sense that this returned, this was also the case in version `0.3.4master-e0daac9`, but versioning got broken after that release and has been fixed with this one. I expected the `winget validate` or the `SandboxTest.ps1` script to catch this.

Reply:
winget validate just validates that the manifest is in the correct format.
SandboxTest.ps1 can be used to install the software and check the Add or Remove Programs metadata fields so its a manual process.
In the future wingetbot will look for incorrect metadata, but thats... the future 😁